src/views/Tweets.vue

```diff
@@ -24,6 +27,9 @@ import Header from '@/components/Header.vue';
 export default class Tweets extends Vue {
   twitter_endpoint : string = `https://us-central1-saot-217513.cloudfunctions.net/tweets`;
   tweets : Array<Object> = [];
+  selected = {};
+  sentiment = "";
```

Comment:
The variables of _sentiment_ and _category_ could both be in the same object variable to bring a more compact and easy to read/use code. Again it does not mean that the code is not functional but these are just some small changes helping us in future development of the code.
`
results = {
      sentiment : String,
      category : String
  };
`

Reply:
This was resolved in 8ef856ccd26642a71e1055d83f45852c0a21d4cd